examples/r2i/top_sort_postprocessing.cc

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2020 RidgeRun, LLC (http://www.ridgerun.com)
+ * All Rights Reserved.
+ *
+ * The contents of this software are proprietary and confidential to RidgeRun,
+ * LLC.  No part of this program may be photocopied, reproduced or translated
+ * into another programming language without prior written consent of
+ * RidgeRun, LLC.  The user is free to modify the source code after obtaining
+ * a software license from RidgeRun.  All source code changes must be provided
+ * back to RidgeRun without any encumbrance.
+ */
+
+#include <algorithm>
+#include <memory>
+#include <vector>
+
+#include <r2i/ipostprocessing.h>
+
+class Example: public r2i::IPostprocessing {
+ public:
+  std::shared_ptr<r2i::IPrediction> Apply(std::shared_ptr<r2i::IPrediction>
+                                          prediction,
+                                          r2i::RuntimeError &error) override {
+    std::shared_ptr<r2i::IPrediction> sorted_prediction;
+
+    error = SortPrediction(prediction);
+
+    sorted_prediction = prediction;
+
+    return sorted_prediction;
+  }
+
+ private:
+  r2i::RuntimeError SortPrediction (std::shared_ptr<r2i::IPrediction>
```

Comment:
I do not understand this part. I think you are missing the label associated with the prediction value in the array.

Reply:
Solved [here](https://github.com/RidgeRun/r2inference/commit/439d98d539d41d6e097af55c762d23e2a4d3a1e1)